zipkin-ui/test/component_data/default.test.js

```diff
@@ -0,0 +1,32 @@
+import {
+  parseApiQuery
+} from '../../js/component_data/default';
+
+chai.config.truncateThreshold = 0;
```

Comment:
This is only needed if we compare really, really deep data structures, like e.g. traces. (For prettier output)